Index: src/it/multi-module-reports-generation-check/moduleB/src/main/java/com/test/moduleB/ClassModuleB.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/it/multi-module-reports-generation-check/moduleB/src/main/java/com/test/moduleB/ClassModuleB.java b/src/it/multi-module-reports-generation-check/moduleB/src/main/java/com/test/moduleB/ClassModuleB.java
new file mode 100644
--- /dev/null	(date 1628714536415)
+++ b/src/it/multi-module-reports-generation-check/moduleB/src/main/java/com/test/moduleB/ClassModuleB.java	(date 1628714536415)
@@ -0,0 +1,12 @@
+package com.test.moduleB;
+
+public class ClassModuleB {
+    public void sayHello(boolean a) {
+        if (a) {
+            System.out.println("if");
+        } else {
+            System.out.println("else");
+        }
+        System.out.println("return");
+    }
+}
Index: src/it/multi-module-reports-generation-check/moduleA/src/main/java/com/test/moduleA/ClassModuleA.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/it/multi-module-reports-generation-check/moduleA/src/main/java/com/test/moduleA/ClassModuleA.java b/src/it/multi-module-reports-generation-check/moduleA/src/main/java/com/test/moduleA/ClassModuleA.java
new file mode 100644
--- /dev/null	(date 1628714375075)
+++ b/src/it/multi-module-reports-generation-check/moduleA/src/main/java/com/test/moduleA/ClassModuleA.java	(date 1628714375075)
@@ -0,0 +1,12 @@
+package com.test.moduleA;
+
+public class ClassModuleA {
+    public void sayHello(boolean a) {
+        if (a) {
+            System.out.println("if");
+        } else {
+            System.out.println("else");
+        }
+        System.out.println("return");
+    }
+}
